src/Drips.sol

```diff
@@ -538,8 +537,8 @@ library Drips {
     /// @notice Calculates the time range in which a receiver is to be dripped to.
     /// This range is capped to provide a view on drips through a specific time window.
     /// @param receiver The drips receiver
-    /// @param updateTime The time when drips are configured
     /// @param defaultEnd The end time of drips without duration
+    /// @param updateTime The time when drips are configured
```

Comment:
These changes are unneeded, they only break the order of the parameters

Reply:
Fixed